tests/functional_tests/serialization_tests.py

```diff
@@ -0,0 +1,79 @@
+from nose.tools import *
+from pymacaroons import Macaroon, Verifier, MACAROON_V1, MACAROON_V2
+from pymacaroons.serializers import JsonSerializer
+
+
+class TestSerializationCompatibility(object):
```

Comment:
I have not kept them up to date, but I had a separate test suite for this: https://github.com/ecordell/macaroon-compatibility